app/src/openxr/cpp/OpenXRInputSource.cpp

```diff
@@ -607,31 +616,34 @@ void OpenXRInputSource::EmulateControllerFromHand(device::RenderMode renderMode,
     }
 
     // Check whether palm of left hand is facing the head, to show navigate-back button
-    bool leftPalmFacesHead = false;
-    if (mHandeness == OpenXRHandFlags::Left) {
-#if defined(OCULUSVR)
-        if (IsHandJointPositionValid(XR_HAND_JOINT_PALM_EXT))
-            leftPalmFacesHead = !mHasAimState;
-#else
-        XrHandJointEXT palmJointIndex = XR_HAND_JOINT_PALM_EXT;
-#if defined(PICOXR)
-        // Pico runtime doesn't provide palm joint info, so we use the wrist instead.
-        palmJointIndex = XR_HAND_JOINT_WRIST_EXT;
+    bool palmFacesHead = false;
+    XrHandJointEXT palmJointIndex = GetHandJointIndexForAim();
```

Comment:
Not sure we need the pañmJointindez, specially if we use the approach I suggested above.

Reply:
OK